### PR TITLE
docs: fix link to example asyncapi document

### PR DESCRIPTION
**Pull request by 1p22geo**
**Description**

- the link in README.md refers to an old filename - the example YAML file in `asyncapi/spec` has been renamed.
- also make it more clear that the link is an example of an AsyncAPI document

---
Old link: [https://raw.githubusercontent.com/asyncapi/spec/master/examples/streetlights-kafka.yml](https://raw.githubusercontent.com/asyncapi/spec/master/examples/streetlights-kafka.yml)
New link: [https://raw.githubusercontent.com/asyncapi/spec/master/examples/streetlights-kafka-asyncapi.yml](https://raw.githubusercontent.com/asyncapi/spec/master/examples/streetlights-kafka-asyncapi.yml)

**Related issue(s)**
none yet

### CODE REVIEW

**Comment by 1p22geo** [comment]
all right, conventional commits, I get it. Renamed the commit from `patch:` to `fix:`

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Generate using CLI
 asyncapi generate fromTemplate <asyncapi.yaml> @asyncapi/markdown-template@1.2.1
 ```
 
-You can replace `<asyncapi.yaml>` with local path or URL pointing to [any AsyncAPI document](https://raw.githubusercontent.com/asyncapi/spec/master/examples/streetlights-kafka.yml).
+You can replace `<asyncapi.yaml>` with local path or URL pointing to any AsyncAPI document like [this one](https://raw.githubusercontent.com/asyncapi/spec/master/examples/streetlights-kafka-asyncapi.yml).
```
Comment (derberg) [inline README.md:32]:
```suggestion
You can replace `<asyncapi.yaml>` with local path or URL pointing to [any AsyncAPI document](https://raw.githubusercontent.com/asyncapi/spec/master/examples/streetlights-kafka-asyncapi.yml).
```

thanks for fixing link, but the alias needs to be old one from the accessibility perspective and making aliases nicer for screen readers. Screen reader for people with reading disability will just say "this one" which will not be much helpful.

**Comment by 1p22geo** [comment]
I'm not really strong on accessibility, but I guess that makes sense...

**Review by derberg — APPROVED** [review]
changed prefix as we do not need to provide a release artefact for readme update

thanks for the fix!

**Pull request merged**